frontend/index.html

```diff
@@ -21,125 +21,115 @@ <h1 class="title">
-                        tiny layer around <a href="https://github.com/hashicorp/vault"
-                            target="_blank">hashicorp vault shamir</a> and golangs <a
-                            href="https://github.com/gtank/cryptopasta/blob/master/encrypt.go" target="_blank">AES
-                            encryption</a>.</p>
-                    <p>If you find any issues, please report them via <a href="https://github.com/simonfrey/s4/issues"
-                            target="_blank">Github issues</a> and if you want to tip me for the work on this project,<a
-                            href="https://simon-frey.com/tip" target="_blank"> feel free to do so !</a></p>
-                </div>
+            <!-- ENCRYPT -->
+            <div class="columns tab" id="encrypt" style="display: block;">
+              <div class="column is-vertical">
+                  <div class="columns is-horizontal is-vcentered">
+                      <div class="column">
+                          Minimum required shares: <input class="input" type="number" value="2" min="2" id="threshold" onChange="doEncrypt(this)" placeholder="Minimum required shares">
+                      </div>
+                      <div class="column ">
+                          Shares: <input class="input" value="3" min="2" type="number" id="shares" onChange="doEncrypt(this)" placeholder="Shares">
+                      </div>
+                      <div class="column ">
+                          <label class="checkbox">
+                              <input type="checkbox" onChange="doEncrypt(this)" class="checkbox" id="useAES"> Use AES(256 bit) for data and
+                              distribute key</label>
+                      </div>
+                  </div>
+                  <div class="columns">
+                      <div class="column">
+                          <textarea class="textarea" id="input" onInput="doEncrypt(this)"
+                              placeholder="Whatever message / binary data you want."></textarea>
+                      </div>
+                  </div>
+
+                  <div class="columns is-horizontal" id="outputs">
+                      <!-- OUTPUTS -->
+                  </div>
+              </div>
             </div>
-            <div class="columns" id="decryptForm" v-if="show">
+
+            <!-- DECRYPT -->
+            <div class="columns tab" style="display:none;" id="decrypt">
                 <div class="column is-vertical">
                     <div class="columns is-horizontal">
                         <div class="column ">
-                            Shares: <input class="input" min="2" type="number" v-model="shares" placeholder="Shares">
+                            Shares: <input class="input" id="decryptShares" value="2" min="2" type="number" onChange="handleShareChange()" placeholder="Shares">
```

Comment:
A little bit of inconsistent style
- line 29-31 call a function passing the value
- line 74 calling the function blank and than calls ` document.getElementById("decryptShares").valueAsNumber;` in decrypt.js 

Wouldn't it be better to use `onChange="handleShareChange(this.value)` here as well?

Reply:
`handleShareChange()` is also called to initialize those fields, so it needs to work without an arg.

The other func that receives `this` utilizes the passed `this` to determine which of the form controls was modified.